### PR TITLE
Added some more terms to the jargon dictionary

### PR DESCRIPTION
**Pull request by Metallion**

### CODE REVIEW

File: docs/content/jargon-dictionary.md [hunk 1]
```diff
@@ -47,6 +57,10 @@ Also known as host or host node.
 
 ### Instance
 
+### Instances network
+
+An L3 network in which [instances](#instance) are started. This can technically be the same network used for the [management line](#management-line-management-network) but on production environments, it is probably a good idea to separate them.
+
 A virtual machine managed by wakame-vdc.
```
Comment (Mopster) [inline docs/content/jargon-dictionary.md]: Needs to be under "Instance"

File: docs/content/jargon-dictionary.md [hunk 2]
```diff
@@ -73,6 +91,11 @@ The NATbox is an optional Wakame-vdc node that provides one to one [network addr
 Refers to a server that a Wakame-vdc process runs on. Can be physical or a VM but not an [instance](#instance).
 
 ## O
+
+### OpenVNet
+
+An full virtual networking implementation using OpenFlow. This project is completely separate from Wakame-vdc but Wakame-vdc can use it to implement virtual networking. This integration is still experimental at this time. [http://openvnet.org](http://openvnet.org)
```
Comment (Mopster) [inline docs/content/jargon-dictionary.md]: 'A full' ? :-) I'd remove the 'experimental' bit, no need for that, it's openvnet related info (should be located ith openvnet docs) and it'll change at some point.

Comment (Metallion) [inline docs/content/jargon-dictionary.md]: More detailed info on OpenVNet could go in the OpenVNet docs but I thought a single mention of it here wouldn't be out of place.

File: docs/content/jargon-dictionary.md (hunk 1 above)
Comment (Metallion) [inline docs/content/jargon-dictionary.md]: Oops. Will fix. :)

**Comment by Mopster** [comment]
:+1:

**Pull request merged**